### DISCUSSION TITLE
accessorKey v8 vs accessor v7

### DISCUSSION THREAD

**Discussion by simplecommerce** [TanStack/table#4161] [Q&A] [answered]
Hi,

I noticed a behavior difference in v8 vs v7 for the accessorKey vs accessor.
In v7 I was able to specify a nested field in my data as such: `description.title` but in v8 this doesn't do anything.
I assume I have to use `accessorFn` to map the sub field in my object.
But is this the intended behavior? 

Thanks!

**Comment by tannerlinsley** [comment, marked as answer]
Yes until we can provide type safety for deep keys, only one level is supported. You can still use the accessorFn to return deep keys

Tanner Linsley
